src/components/LoginContainer.js

```diff
@@ -27,26 +37,17 @@ class LoginContainer extends Component {
     }
 
     _email() {
-        return (i => {
-            this.email = i;
+        return (ref => {
+            this.email = ref;
         });
     }
 
     _password() {
-        return (i => {
-            this.password = i;
+        return (ref => {
```

Comment:
I also used `i` in every other place where we had string refs before, so this is the outlier now